### DISCUSSION TITLE
BTO Acoustic Pipeline

### DISCUSSION THREAD

**Discussion by glenvine** [mcguirepr89/BirdNET-Pi#1200] [General]
Occasionally I use the Pipeline to verify IDs. However BTO have a syntax for file names requiring the date and time to be added in a specific format as either a prefix or suffix.
https://www.bto.org/our-science/products-and-technologies/bto-acoustic-pipeline/support-hub/advice-commonly-used
"Other digital audio recorders"
Can I ask if there is a way the existing naming convention can be changed, or is this something that would need to be done at code level?
Thank you. MikeH

**Comment by OliNau** [comment]
Hi there,
Since this repository is not being updated any more, you might want to add your request as an issue to a forks of this project (like [this one](https://github.com/Nachtzuster/BirdNET-Pi))

In the meantime, I've written a small script to do the filename conversion. It's unclear to me if your 'BirdNET-Pi filenames' are the same as mine. If they are, you could use [this script](https://github.com/OliNau/birdnet-Pi-BPO-filename-converter) to convert.
Kind regards,

**Reply by glenvine** [reply]
Thank you for the suggestion and the script.
